src/Illuminate/Contracts/Database/MigrationEvent.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Illuminate\Contracts\Database;
```

Comment:
Put this under the `Illuminate\Contracts\Database\Events` namespace. Also move the file to the correct directory.

Reply:
Done